### PR TITLE
Generate a CSV file

### PR DESCRIPTION
**Pull request by nelhage**
I wanted to use this data as a source for plots. https://observablehq.com/@nelhage/vote-count-exploration is using a CSV I generated locally, and I'd love to be able to point it at the github pages live one.

The resulting CSV isn't *ideal* for machine consumption; I'm happy to be told to clean it up before you'd accept this.

### CODE REVIEW

File: print-battleground-state-changes [hunk 1]
```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 
-import os
+import csv
 import collections
 import datetime
 import git
 import itertools
+import os
```
Comment (thomasdesr) [inline print-battleground-state-changes:8]: nit: Unused import
```suggestion
```

Comment (nelhage) [reply print-battleground-state-changes:8]: This was added for the caching, which uses `os.path`, and then I guess I sorted it into the right place wrt the other imports when I revisited the import list for the CSV generation.

Comment (thomasdesr) [reply print-battleground-state-changes:8]: Oh, I'm blind. Sorry X.X

Comment (nelhage) [reply print-battleground-state-changes:8]: No worries -- the unrelated hunk in this PR was definitely confusing.

**Pull request merged**